spring-cloud-azure-keyvault-config/src/main/java/com/microsoft/azure/spring/cloud/keyvault/config/KeyVaultConfigProperties.java

```diff
@@ -36,5 +36,5 @@
 
     private String name;
 
-    private String activeProfile;
+    private String activeProfiles;

```

Comment:
Why plural? Activate multiple profiles once?

Reply:
Yes. Providing possible overrides for the default `spring.profiles.active`.